boards/arm/cc1352r1_launchxl/cc1352r1_launchxl.dts

```diff
@@ -59,10 +59,25 @@
 			label = "Push button 2";
 		};
 	};
+
+	idle-states {
```

Comment:
So, in [Linux](https://www.kernel.org/doc/html/v5.0/admin-guide/pm/cpuidle.html)...

```
Modern processors are generally able to enter states in which the execution of a program is suspended and instructions belonging to it are not fetched from memory or executed. Those states are the idle states of the processor.

Since part of the processor hardware is not used in idle states, entering them generally allows power drawn by the processor to be reduced and, in consequence, it is an opportunity to save energy.

CPU idle time management is an energy-efficiency feature concerned about using the idle states of processors for this purpose.
```

Reply:
Yes.  What does that have to do with our need to describe power states that aren't idle states?

Reply:
> Yes. What does that have to do with our need to describe power states that aren't idle states?

A lot. idle states here applies to all states that can be transitioned to when the cpu is idle, it is not referring to the idle power state.

Reply:
OK.  So then we need a `power-states` that provides all the states, regardless of whether they can be used when transitioning to idle?  Because the power management infrastructure should allow us to move between them, not just pop into and out of a generic idle condition.

Reply:
Another way: We know which of the power states are capable of being entered when the CPU is idle.  It's the ones that aren't `PM_STATE_ACTIVE` and `PM_STATE_RUNTIME_IDLE`.   That's indicated within each state by (currently) `idle-state-name`.